back/app/controllers/web_api/v1/internal_comments_controller.rb

```diff
@@ -0,0 +1,196 @@
+# frozen_string_literal: true
+
+class WebApi::V1::InternalCommentsController < ApplicationController
+  before_action :set_post_type_id_and_policy, only: %i[index create]
+  before_action :set_comment, only: %i[children show update mark_as_deleted destroy]
+  # skip_before_action :authenticate_user # Not sure if this is needed, or even a good idea. TBD
```

Comment:
I remember we added this line to many controllers some time ago, but don't remember why.

I am assuming it's better to not include this line, if possible.

Reply:
✅ Removed